nvtabular/inference/triton/model.py

```diff
@@ -24,306 +24,45 @@
-            transformed, kind = _transform_tensors(input_tensors, self.workflow.output_node)
-
-            # if we don't have tensors in numpy format, convert back so that the we can return
-            # to triton
-            if kind != Supports.CPU_DICT_ARRAY:
-                transformed, kind = convert_format(transformed, kind, Supports.CPU_DICT_ARRAY)
-
-            # convert to the format expected by the DL models
-            if self.output_model == "hugectr":
-                response = self._transform_hugectr_outputs(transformed)
-            elif self.output_model == "pytorch":
-                response = self._transform_pytorch_outputs(transformed)
-            else:
-                response = self._transform_outputs(transformed)
-            responses.append(response)
-
-        return responses
-
-    def _transform_outputs(self, tensors):
-        """transforms outputs for both pytorch and tensorflow"""
-        output_tensors = []
-        for name, value in tensors.items():
-            if isinstance(value, tuple):
-                # convert list values to match TF dataloader
-                values = value[0].astype(self.output_dtypes[name + "__values"])
-                values = values.reshape(len(values), 1)
-                output_tensors.append(Tensor(name + "__values", values))
-
-                offsets = value[1].astype(self.output_dtypes[name + "__nnzs"])
-                nnzs = offsets[1:] - offsets[:-1]
-                nnzs = nnzs.reshape(len(nnzs), 1)
-                output_tensors.append(Tensor(name + "__nnzs", nnzs))
-            else:
-                d = value.astype(self.output_dtypes[name])
-                d = d.reshape(len(d), 1)
-                output_tensors.append(Tensor(name, d))
-        return InferenceResponse(output_tensors)
-
-    def _transform_pytorch_outputs(self, tensors):
-        output_tensors = []
-        for output_name, cols in self.column_types.items():
-            output_tensors.append(
-                Tensor(
-                    output_name,
-                    _convert_to_pytorch(cols["columns"], tensors, cols["dtype"]),
-                )
-            )
-
-        return InferenceResponse(output_tensors)
-
-    def _transform_hugectr_outputs(self, tensors):
-        output_tensors = []
-        if "conts" in self.column_types:
-            output_tensors.append(
-                Tensor(
-                    "DES",
-                    _convert_to_hugectr(self.column_types["conts"], tensors, np.float32),
-                )
-            )
+        model_kind = args["model_instance_kind"]
+        model_config = json.loads(args["model_config"])
+        output_model = model_config["parameters"]["output_model"]["string_value"]
+
+        if output_model == "hugectr":
+            runner_class = HugeCTRWorkflowRunner
+        elif output_model == "pytorch":
+            runner_class = PyTorchWorkflowRunner
+        elif output_model == "tensorflow":
```

Comment:
I'm not entirely sure that `"tensorflow"` is the right value here. It might just be `"model"`?